README.md

```diff
@@ -1,2 +1,19 @@
 # HW01
 Edit the readme
+
+## About Me
+My name is Luke Dyer, and I grew up in [*Jefferson City, MO*](https://en.wikipedia.org/wiki/Jefferson_City,_Missouri). Growing up, I helped my mom, who has a license through the Conservation Department, raise **all kinds** of wild animals. From bats, to deer, to racoons, we did it all. This inspired my love for science and led me to earn a degree in biochemistry at the Missouri University of Science and Technology, in Rolla, MO. This fall, I will be moving to Chicago to start my PhD in Cell and Molecular Biology at the University of Chicago.
```

Comment:
That's amazing! I've always wanted to work with wildlife conservation organizations. Also, the link to the Wiki page of *Jefferson City* is very useful.